src/reducers/filter.js

```diff
@@ -0,0 +1,18 @@
+const filter = (state, action) => {
+  switch (action.type) {
+    case 'CHANGE_FILTER': {
+      return {
+        ...state,
+        category: action.payload.filter,
+      };
+    }
+    default: {
+      return {
+        ...state,
+        category: 'All',
+      };
+    }
+  }
+};
+
+export default filter;
```

Comment:
- [ ] Since the Filter reducer state is an object with only one value, category, there is no need to destructuring the state object. Here we can remove the `...state` from the 2 cases. Let's change this before merging the branch

Reply:
Ok, thanks